MapboxSearch/gradle/versions.gradle

```diff
@@ -19,7 +19,6 @@ ext {
 
     ktlint_version = '0.39.0'
     okhttp_version = '4.9.0'
```

Comment:
Should we remove `okhttp_version` and `okhttp_mock_version` dependencies as well?

Reply:
No, some parts of OkHttp are still needed for mock web server in tests.